src/node.cc

```diff
@@ -2914,9 +2914,6 @@ void SetupProcessObject(Environment* env,
   env->SetMethod(process, "_setupNextTick", SetupNextTick);
   env->SetMethod(process, "_setupPromises", SetupPromises);
   env->SetMethod(process, "_setupDomainUse", SetupDomainUse);
-
-  // pre-set _events object for faster emit checks
-  process->Set(env->events_string(), Object::New(env->isolate()));
```

Comment:
It will be possible with v8 4.5: https://github.com/v8/v8-git-mirror/commit/395fa8ba2439f7e768376f7512553d66b67d1c87